tests/docs/test_docs.py

```diff
@@ -13,11 +13,13 @@ def test_html(self):
         doctrees_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['doctrees'])
         html_path = os.path.sep.join(self.path_to_docs.split(os.path.sep) + ['_build']+['html'])
 
-        check = subprocess.call(["sphinx-build", "-nW", "-b", "html", "-d",
-            "{0!s}".format((doctrees_path)) ,
-            "{0!s}".format((self.path_to_docs)),
-            "{0!s}".format((html_path))])
-        assert check == 0
+        check = subprocess.Popen(("sphinx-build", "-nW", "-b", "html", "-d",
+            "{0!s}".format(doctrees_path),
+            "{0!s}".format(self.path_to_docs),
+            "{0!s}".format(html_path)),
+            stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = check.communicate()
+        assert check == 0, 'Test failed with stdout:\n{}\n and stderr:\n{}\n'.format(out,err)
```

Comment:
I agree, the second formatted one is much easier to read.  
I was running the test locally and I wasn't getting good error reporting. It looks like travis grabs them and prints them out and by redirecting them it doesn't happen. 
I vote for the old version.